PAO-Project/src/Services/Student_S.java

```diff
@@ -0,0 +1,83 @@
+package Services;
+
+import Utilizator.Student;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Student_S
+{
+    private List<Student> studenti = new ArrayList<>();
+    private static Student_S instance;
+
+    private Student_S(){}
+
+    public static Student_S get_Instance(){
+        if(instance == null){
+            instance = new Student_S();
+        }
+        return instance;
+    }
+
+    public List<Student> get_Studenti()
```

Comment:
[Improvement] nu e nevoie sa aiba cuvantul "student" in numele metodei, e clar ce cauta, modificam, creem sau stergem fiindca apelam aceste metode din serviciul pentru studenti si e redundant. La fel si pentru celelalte servicii